pkg/koordlet/metriccache/metric_resources.go

```diff
@@ -29,4 +29,24 @@ var (
 	PodCPUThrottledMetric       = defaultMetricFactory.New(PodMetricCPUThrottled).withPropertySchema(MetricPropertyPodUID)
 	ContainerCPUUsageMetric     = defaultMetricFactory.New(ContainerMetricCPUUsage).withPropertySchema(MetricPropertyContainerID)
 	ContainerCPUThrottledMetric = defaultMetricFactory.New(ContainerMetricCPUThrottled).withPropertySchema(MetricPropertyContainerID)
+
+	// CPI
+	ContainerCPICycleMetric       = defaultMetricFactory.New(ContainerMetricCPICycle).withPropertySchema(MetricPropertyPodUID, MetricPropertyContainerID)
+	ContainerCPIInstructionMetric = defaultMetricFactory.New(ContainerMetricCPIInstruction).withPropertySchema(MetricPropertyPodUID, MetricPropertyContainerID)
+
+	// PSI
```

Comment:
better to define ContainerPSI with property: containerID, podUID, resource(CPU, Memory, IO), precision(10, 60, 300), degree(full, some)